wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -319,21 +321,32 @@ def table_header_label(context, label=None, sortable=True, ordering=None, sort_c
         # currently ordering forwards on this column; link should change to reverse ordering
         url = querystring(context, **{sort_param: reverse_sort_field})
         classname = "icon icon-arrow-down-after teal"
+        ordering_label = _('descending')
 
     elif ordering == reverse_sort_field:
         # currently ordering backwards on this column; link should change to forward ordering
         url = querystring(context, **{sort_param: sort_field})
         classname = "icon icon-arrow-up-after teal"
+        ordering_label = _('ascending')
 
     else:
         # not currently ordering on this column; link should change to forward ordering
         url = querystring(context, **{sort_param: sort_field})
         classname = "icon icon-arrow-down-after"
+        ordering_label = _('ascending')
+
+    title_tag = _('Sort the order of child pages within \'')
+    title_tag += parent_page_title
+    title_tag += "%s" % (_('\' by \''))
+    title_tag += label
+    title_tag += "%s" % (_('\' in '))
+    title_tag += "%s" % (ordering_label)
+    title_tag += "%s" % (_(' order.'))
 
     return format_html(
         # need whitespace around label for correct positioning of arrow icon
-        '<a href="{url}" class="{classname}"> {label} </a>',
-        url=url, classname=classname, label=label
+        '<a href="{url}" class="{classname}" title="{title_tag}"> {label} </a>',
+        url=url, classname=classname, label=label, title_tag=title_tag
```

Comment:
Do we want this to be a `title` attribute or `visuallyhidden` text? `title` sounds like a good idea but I’d like to hear your thoughts on this @helenb.

Reply:
It could be either - but it was simpler to add as a separate title tag rather than try and work out the translated text along with some of it being visually hidden - I don't know how to do the equivalent of blocktrans in python code.